vsce/Makefile

```diff
@@ -0,0 +1,17 @@
+# We can derive Reach's Visual Studio Code extension's
+# version using the VERSION file, which can be found
+# in the parent folder.
+include ../VERSION
```

Comment:
Add
```
ROOT = ..
```
And use
```
include $(ROOT)/VERSION
```
in case we ever move this directory

Reply:
That makes sense. 87d0eca should address this.